examples/advi.py

```diff
@@ -55,7 +55,7 @@ def batch_elbo(logprob, rng, params, num_samples):
 
 # ========= Helper function for plotting. =========
 
-@partial(jit, static_argnums=(0, 1, 2, 4))
+@partial(jit, static_argnums=(0, 1, 2))
 def mesh_eval(func, x_limits, y_limits, params, num_ticks=101):
```

Comment:
The 4th positional argument, `num_ticks`, has a default value, and is never explicitly supplied during invocation. Hence the error. 

I took a look at [f_jitted]() but don't see a way to handle this scenario (i.e., include the default args)

Reply:
Oh hold on I may be able to do this